lib/services/dnsManagement2/README.md

```diff
@@ -0,0 +1,9 @@
+# Microsoft Azure SDK for Node.js - DNS Management    
+    
+This project provides a Node.js package that makes it easy to manage Microsoft Azure DNS.     
+    
+## How to Install    
+    
+```bash    
+npm install azure-arm-dns    
+```  
```

Comment:
will add this later.. currently not changing this since it is the same for prev release as well